lib/board.rb

```diff
@@ -0,0 +1,48 @@
+require 'pry'
+class Board
+  attr_accessor :cells
+
+  def initialize
+    @cells = Array.new(9, " ") 
+  end
+
+  def reset!
+    @cells = Array.new(9, " ") 
+  end
+
+  def display
+    puts "  #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+    puts " ----------- "
+    puts "  #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+    puts " ----------- "
+    puts "  #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+  end
+
+  def position(pos)
+    @cells[pos.to_i - 1]
+  end
+
+  def full?
+    @cells.include?(" ") ? false : true
+  end
+
+  def turn_count
+    9 - @cells.count(" ")
+  end
+
+  def taken?(position)
+    @cells[position.to_i - 1] == " " ? false : true
```

Comment:
Use the implicit boolean return value of the expression instead of adding the ternary. Additionally, you can reverse (or negate) the boolean with the `!` (bang) operator.

`@cells[position.to_i - 1] != " "`